linux/ibus-keyman/src/KeymanSystemServiceClient.cpp

```diff
@@ -111,6 +112,23 @@ KeymanSystemServiceClient::CallOrderedOutputSentinel() {
   }
 }
 
+void KeymanSystemServiceClient::Ping() {
```

Comment:
Maybe a little more descriptive
```suggestion
void KeymanSystemServiceClient::PingKeymanSystemService() {
```

Reply:
Currently this matches the method names on the DBus interface, so I think I leave it as `Ping`. But thinking of it, `CallOrderedOutputSentinel` isn't probably that good of a method name, but I can't think of  a better one.